openapi/customervoice/customervoice.yaml

```diff
@@ -231,8 +500,121 @@ components:
                       type: string
                       default: template
                       readOnly: true
+    templates:
+      title: Templates
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Templates that can be used for emails and other messaging services.
+      type: object
+      x-examples:
+        Example Template:
+          type: templates
+          id: 'AG-abc123:email:TEM-d2f4cf1aa7ae480bbf0a2da67dxxxxxx'
+          attributes:
+            name: My Preferred Sites Review Request
+            content: |-
+              Hi {{.customerName}},
+              <br>
+              <br>
+              Thank you for your business! We'd love to know more about your latest experience with us.
+              <br>
+              <br>
+              {{.buttonUrl}}
+              <br>
+              <br>
+              Thank you,
+              <br>
+              John Doe
+            metadata:
+              subject: Thank you for choosing us!
+              replyEmail: jondoe@mail.com
+            locale: en-US
+            type: email
+          relationships:
+            businessLocation:
+              data:
+                id: AG-12345
+      x-lifecycle:
+        status: trustedTester
+      x-tags:
+        - Templates
+      properties:
+        id:
+          type: string
+          example: AG-abc123:email:TEM-d2f4cf1aa7ae480bbf0a2da67dxxxxxx
+          readOnly: true
+        type:
+          type: string
+          default: templates
+          enum:
+            - templates
+          readOnly: true
+        attributes:
+          type: object
+          properties:
+            name:
+              type: string
+              description: A user friendly description for the template.
+              example: My Preferred Sites Review Request
+            metadata:
+              type: object
+              description: A JSON object containing key-value pairs that can be used to store additional information about the template.
+              example:
+                subject: Thank you for choosing us!
+                replyEmail: jondoe@mail.com
```

Comment:
How will clients know what keys are valid for metadata? Are they well defined for email and SMS? I'd like to pull the keys up to top level attributes if we can.